lib/cli.js

```diff
@@ -24,6 +25,8 @@ updateNotifier({ pkg }).notify();
 
 if (argv.v) {
   console.log(`v${pkg.version}`);
+} else if (!isGit()) {
```

Comment:
Do we use `{}` now for one-line stmt or not? I would suggest that we use the same structure in the whole project.